tests/security/test_reentrancy.py

```diff
@@ -11,7 +11,7 @@
 async def reentrancy_mock():
     starknet = await Starknet.empty()
     contract = await starknet.deploy(
-        "tests/mocks/reentrancy_mock.cairo",
+        contract_class=get_contract_class("reentrancy_mock"),
```

Comment:
how was this working?

Reply:
Without the `contract_class` kwarg, you mean?

Reply:
yes